projects/packages/assets/src/class-assets.php

```diff
@@ -651,7 +651,7 @@ private static function init_domain_map_hooks( $domain, $firstcall ) {
 	 */
 	public static function filter_gettext( $translation, $text, $domain ) {
 		if ( $translation === $text ) {
-			// phpcs:ignore WordPress.WP.I18n
+			// phpcs:ignore WordPress.WP.I18n -- we're including text domains from the most recent dependency package versions, of which there are generally a fair number, to ensure we have the most recent translations.
```

Comment:
I'd have simplified this a bit, and added a link to https://developer.wordpress.com/2022/01/06/wordpress-plugin-i18n-webpack-and-composer/#problem-6-text-domains-in-composer-packages
```suggestion
			// phpcs:ignore WordPress.WP.I18n -- This is a filter hook to map the text domains from our Composer packages to the domain for a containing plugin. See https://wp.me/p2gHKz-oRh#problem-6-text-domains-in-composer-packages
```

Reply:
Good idea, I'll follow up.